index.js

```diff
@@ -31,7 +31,8 @@ const preserveCamelCase = input => {
 
 module.exports = (input, options) => {
 	options = Object.assign({
-		pascalCase: false
+		pascalCase: false,
+		ignoreDotNotation: false
```

Comment:
I'd rather have an `ignore` option in which the user can choose which characters they want to preserve.

Reply:
I agree I prefer not continuing to patch with more ignore cases.